### PR TITLE
Shadowlands templates

### PR DESCRIPTION
**Pull request by mrbuds**
# Description

- [x] remove azerite stuff
- [x] priest skills update
- [ ] more classes
- [ ] covenant abilities

### CODE REVIEW

File: WeakAurasTemplates/TriggerTemplatesData.lua [hunk 1]
```diff
@@ -5185,6 +5185,7 @@ if not WeakAuras.IsClassic() then
     for className, class in pairs(templates.class) do
       for specIndex, spec in pairs(class) do
         for _, section in pairs(spec) do
+          if section.args then
```
Comment (InfusOnWoW) [inline WeakAurasTemplates/TriggerTemplatesData.lua:5196]: Don't be afraid of fixing the indentation

Comment (InfusOnWoW) [reply WeakAurasTemplates/TriggerTemplatesData.lua:5196]: Ah you did in the follow up commit

**Comment by InfusOnWoW** [comment]
I don't think we need to fix all classes in one PR. It's good enough if you are happy enough with priests, we should just merge it.

**Comment by Stanzilla** [comment]
Do we want to kill the templates right away? Because we're publishing this version of WA with the prepatch which still does have Azerite and Essences.

**Comment by InfusOnWoW** [comment]
Yes, the few people that want to create auras for essences can do it manually.

**Comment by mrbuds** [comment]
i'v rebased, and i agree that it can be merged now and do more updates later

**Pull request merged**